builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -197,6 +201,19 @@ func resourceAwsRoute53ZoneRead(d *schema.ResourceData, meta interface{}) error
 func resourceAwsRoute53ZoneUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).r53conn
 
+	if d.HasChange("comment") {
```

Comment:
Shouldn't `comment` be set as partial + `d.Partial(true)` used? It's one API call (out of two) that can fail, right?
Before it was only 1 call for `tags`, so I don't think `d.SetPartial("tags")` was actually necessary before.

btw. I think `d.SetPartial("tags")` in the place where it is now (`else` block) is completely useless (putting away the fact that `d.Partial(true)` isn't called), but maybe I'm just misunderstanding the way this works?